webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
This seems to be imported but not used.

Reply:
I swear it broke on my machine when i tried to remove this earlier
But yes its completely useless.